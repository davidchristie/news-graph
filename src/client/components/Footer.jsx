import React from 'react'

export default class Footer extends React.Component {
  render () {
    return (
      <footer className='bd-footer text-muted' style={{marginTop: '100px'}}>
        <div className='container'>
          <p>
            <a href='https://github.com/davidchristie/news-graph'>GitHub</a>
          </p>
          <p>
            Powered by <a href='https://newsapi.org'>NewsAPI.org</a>
          </p>
        </div>
      </footer>
    )
  }
}
