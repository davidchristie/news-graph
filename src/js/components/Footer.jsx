import React from 'react'

export default class Footer extends React.Component {
  render() {
    return (
      <footer className="bd-footer text-muted" style={{marginTop: '100px'}}>
        <div className="container">
          <a href="https://github.com/davidchristie/news-graph">GitHub</a>
          <p>
            Insert footer content here.
          </p>
        </div>
      </footer>
    )
  }
}
