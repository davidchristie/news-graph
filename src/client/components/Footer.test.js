/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Footer from './Footer'

test('Footer renders correctly', () => {
  const wrapper = shallow(<Footer />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
