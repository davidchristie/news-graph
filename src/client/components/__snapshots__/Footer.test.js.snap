exports[`test Footer renders correctly 1`] = `
<footer
  className="bd-footer text-muted"
  style={
    Object {
      "marginTop": "100px",
    }
  }>
  <div
    className="container">
    <p>
      <a
        href="https://github.com/davidchristie/news-graph">
        GitHub
      </a>
    </p>
    <p>
      Powered by 
      <a
        href="https://newsapi.org">
        NewsAPI.org
      </a>
    </p>
  </div>
</footer>
`;
